lazyflow/operators/ioOperators/ioOperators.py

```diff
@@ -559,51 +562,58 @@ def setupOutputs(self):
         self.outputs["WriteImage"].meta.shape = (1,)
         self.outputs["WriteImage"].meta.dtype = object
 
-        self.f = self.inputs["hdf5File"].value
-        hdf5Path = self.inputs["hdf5Path"].value
+        self.f = self.inputs["h5N5File"].value
+        h5N5Path = self.inputs["h5N5Path"].value
 
         # On windows, there may be backslashes.
-        hdf5Path = hdf5Path.replace('\\', '/')
+        h5N5Path = h5N5Path.replace('\\', '/')
 
-        hdf5GroupName, datasetName = os.path.split(hdf5Path)
-        if hdf5GroupName == "":
+        h5N5GroupName, datasetName = os.path.split(h5N5Path)
+        if h5N5GroupName == "":
             g = self.f
         else:
-            if hdf5GroupName in self.f:
-                g = self.f[hdf5GroupName]
+            if h5N5GroupName in self.f:
+                g = self.f[h5N5GroupName]
             else:
-                g = self.f.create_group(hdf5GroupName)
+                g = self.f.create_group(h5N5GroupName)
 
         dataShape=self.Image.meta.shape
         self.logger.info( "Data shape: {}".format(dataShape))
```

Comment:
```suggestion
         self.logger.info( f"Data shape: {dataShape}")
```

Ditto for other string formats.